app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/conversationSearchBar.tsx

```diff
@@ -54,6 +55,10 @@ export const ConversationSearchBar = ({
     debouncedSetSearch(search);
   }, [search]);
 
+  useHotkeys(["ctrl+k", "meta+k"], () => {
```

Comment:
```suggestion
  useHotkeys("mod+k", () => {
```

I think this will use cmd on Mac and ctrl on Windows - don't suppose you have a Windows PC to verify?

Reply:
Yeah that worked, turns out you were [right](https://github.com/JohannesKlauss/react-hotkeys-hook/blob/b9e9dd39abec4f62d49cc92440b238bef690a1bf/packages/react-hotkeys-hook/src/lib/validators.ts#L96)! I was able to verify this with a Windows VM.